.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,95 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug", "triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Before filling out this form,
+        please review the (Cluster and OS Pre-Requisites)[https://github.com/openshift/windows-machine-config-operator/blob/master/docs/wmco-prerequisites.md],
+        as well as the (BYOH Instance Pre-requisites)[BYOH Instance Pre-requisites].
+  - type: checkboxes
+    id: community-check
+    attributes:
+      label: Community Issue
+      description: |
+        Github issues are for those using the community distribution of WMCO.
+        Support for users with a subscription should be requested through the Red Hat Customer Portal.
+      options:
+        - label: I am using the community version of WMCO
+          required: true
+  - type: dropdown
+    id: cluster-type
+    attributes:
+      label: Version
+      description: Are you using an OKD or OCP cluster?
+      options:
+        - OKD
+        - OCP
+    validations:
+      required: true
+  - type: input
+    id: cluster-version
+    attributes:
+      label: Cluster Version
+      description: What is the version of your cluster?
+      placeholder: 4.12
+    validations:
+      required: true
+  - type: dropdown
+    id: platform
+    attributes:
+      label: Platform
+      description: What platform is your cluster running on?
+      options:
+        - AWS
+        - Azure
+        - Bare metal
+        - GCP
+        - Platform agnostic (type=none)
+        - vSphere
+    validations:
+      required: true
+  - type: input
+    id: wmco-version
+    attributes:
+      label: WMCO Version
+      description: What version of WMCO are you using?
+      placeholder: 6.0.0
+    validations:
+      required: true
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: What did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: reproduction-steps
+    attributes:
+      label: Steps to reproduce the issue
+      description: Is this consistently reproducable? If not how often does it occur?
+    validations:
+      required: true
+  - type: textarea
+    id: workaround
+    attributes:
+      label: Do you have a workaround for this issue?
+    validations:
+      required: false
+  - type: markdown
+    attributes:
+      value: |
+        Please attach any logs and relevant screenshots in a comment.
+        At minimum, please include the output of the following
+        * oc get network.operator cluster -o yaml
+        * oc logs deployment/windows-machine-config-operator -n openshift-windows-machine-config-operator
+        * If relevant: oc get machineset <windows_machineSet_name> -n openshift-machine-api -o yaml
```

Comment:
Shouldn't we also ask for the 
- windows-instances ConfigMap?
- Cluster config that will tell us if proxy is being used?

Reply:
Yep, added a line for the CM, and a required field for proxy information